website/docs/framework-docs/getting-started/quickstart.md

```diff
@@ -65,7 +75,7 @@ Add the following dependency to your `pom.xml` file:
 <dependency>
   <groupId>io.github.boykaframework</groupId>
   <artifactId>boyka-framework</artifactId>
-  <version>2.0.0</version>
+  <version>2.1.0</version>
```

Comment:
Provide the Maven central dependency link here, instead of hardcoded dependency, as with every release it needs to be updated